fastlane/Fastfile

```diff
@@ -216,9 +216,23 @@ lane :test_ui do |options|
     testplan: "StreamChatUITestPlan",
     configuration: "Debug",
     clean: true,
-    devices: options[:device],
-    code_coverage: false
-    )
+    devices: options[:device]
+  )
+end
+
+desc "Runs ui tests in Release config"
+lane :test_ui_release do |options|
+
+  setCIEnvironmentVariable("../Tests/StreamChatUITests/StreamChatUITestPlan.xctestplan")
```

Comment:
Can we also add the `if !is_ci {}` check to this action in this PR?

Reply:
Yes makes sense 👍

Reply:
We have a whole separate ticket for Fastfile changes https://stream-io.atlassian.net/secure/RapidBoard.jspa?rapidView=52&projectKey=CIS&modal=detail&selectedIssue=CIS-233

Reply:
We will add this change in the Fastfile cleanup then.